src/components/team/css.module.scss

```diff
@@ -0,0 +1,38 @@
+#outer{
+    width: 100%;
+    text-align: center;
+}
+
+.inner{
+    display: inline-block;
+    background: rgb(179, 229, 248);
+}
+
+.roundedEdges{
```

Comment:
Noticed you use this more than once for different classes this should be in the general scss file. Try to always avoid duplicating code.

Reply:
Gotchu